src/Modules/Pantry/EventNearByComponent.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect, Fragment } from 'react';
+// import { useForm } from 'react-hook-form';
+import { useParams, withRouter } from 'react-router-dom';
+// import { useDispatch } from 'react-redux';
+// import { ProgressBar } from 'react-bootstrap';
+// import SearchComponent from '../General/SearchComponent';
```

Comment:
lets not merge unused/commented code here

Reply:
Done